src/App.vue

```diff
@@ -86,6 +86,10 @@ export default {
         e.preventDefault();
       }
     });
+
+    const threeJSModuleScript = document.createElement('script');
+    threeJSModuleScript.setAttribute('src', '3DViewer/libtest_lib.js');
+    document.head.appendChild(threeJSModuleScript);
```

Comment:
Is your goal to load this async? You could also use `<script src="3DViewer/libtest_lib.js" async></script>` in the index.html file

Reply:
I had some issue where adding a `<script>` tag caused the build to not inject the other scripts.  I'm not sure why it happened, but it's not happening any more, so I'll update that.

Reply:
I discovered some new and exciting issues when using a `<script>` tag - namely, the build fails because something in webpack is trying to modify the `src` attribute.

I'm going to stick with the JS approach for now.